README.go.md

```diff
@@ -0,0 +1,10 @@
+# iroh-go
+
+Instructions on how to use `iroh-ffi/iroh-go` in your project!
+
+```
+$ go get github.com/n0-computer/iroh-ffi/iroh-go/iroh
+$ git submodule add https://github.com/n0-computer/iroh-ffi.git extern/iroh-ffi
+$ make -f extern/iroh-ffi/InstallGo
```

Comment:
does this require a working rust toolchain?

Reply:
no, this should download the libiroh file from our releases page & slot it into the expected location. No need to have rust.

Reply:
great